RecipeApp/Profile/DetentsViewController.swift

```diff
@@ -0,0 +1,89 @@
+// DetentsViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран для всплывающего экрана с бонусами
+final class DetentsViewController: UIViewController {
+    // MARK: - Visual Components
+
+    private let xmarkImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "xmark")
```

Comment:
```suggestion
        imageView.image = UIImage(named: "")
```
и ниже все тоже в константы